packages/esm-patient-chart-app/src/ui-components/patient-chart/patient-chart.component.tsx

```diff
@@ -27,37 +27,34 @@ const PatientChart: React.FC<RouteComponentProps<PatientChartParams>> = ({
   useVisitDialog(patientUuid);
 
   return (
-    <>
-      <SideMenu />
```

Comment:
@FlorianRappl This is the Side Menu component that I removed from the patient-chart-component.